spec/system/support_interface/revert_rejection_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.feature 'Revert an accidental rejection' do
+  include DfESignInHelpers
+
+  scenario 'View structured reasons for rejection', with_audited: true do
```

Comment:
nit: worth correcting the scenario here so the commit makes sense in isolation (can see it's corrected later on)